src/lua/LuaPlayer.cpp

```diff
@@ -700,7 +700,8 @@ static int l_player_get_cruise_direction(lua_State *l)
 	Player *player = LuaObject<Player>::CheckFromLua(1);
 	auto mode = player->GetPlayerController()->GetCruiseDirection();
 	auto str = EnumStrings::GetString("CruiseDirection", mode);
-	assert(str && "Wrong get cruise direction");
+	if (!str)
+		luaL_error(l, "Wrong get cruise direction");
```

Comment:
This message (in `GetCruiseDirection()`) may be acceptable to keep as an assert, as Lua context is unable to cause an error by supplying an invalid parameter.